### ISSUE TITLE
Fix PMD configuration for JUnit 5

### ISSUE THREAD

**Issue by JWGmeligMeyling** [SERG-Delft/jpacman-template#31]
Since the migration to JUnit 5, the warnings generated for JUnit by PMD do not make any sense anymore. Besides the default JUnit rules provided by PMD ([1](https://github.com/SERG-Delft/jpacman-template/blob/master/pmd-rules.xml#L19-L69) and [2](http://pmd.sourceforge.net/pmd-4.3.0/rules/index.html#JUnit_Rules), we also included one of our own PMD-rules to warn students for invalidly catching exceptions in tests.

Continues at https://github.com/SERG-Delft/jpacman-solution/issues/13

**Comment by avandeursen** [comment]
Yes, you are right. I had done this for the framework, but forgot to do this for the template.

Just copying the framework's pmd config file to the template would have worked too.
